apps/dashboard/app/helpers/recently_used_apps_helper.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# Helper methods for RecentlyUsedApps widget
+module RecentlyUsedAppsHelper
+  # Returns the 4 most recently used interactive applications based on their cache file
+  # @return [Array<BatchConnect::App>] The 4 most recently used apps
+  def recently_used_apps
+    cache_files = BatchConnect::Session.cache_root.children.sort_by do |pathname|
+                    File.mtime(pathname)
+                  end.reverse.map { |pathname| pathname.basename.to_s }.slice(0, 4)
```

Comment:
Let's try to keep uniformity here in these closures.

```suggestion
                  end.reverse.map do |pathname| 
                    pathname.basename.to_s 
                   end.slice(0, 4)
```

Reply:
fixed